eslint.config.mjs

```diff
@@ -0,0 +1,95 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-licensing-options
+ */
+
+import globals from 'globals';
+import { defineConfig } from 'eslint/config';
+import ckeditor5Rules from 'eslint-plugin-ckeditor5-rules';
+import ckeditor5Config from 'eslint-config-ckeditor5';
+import pluginVue from 'eslint-plugin-vue';
+import ts from 'typescript-eslint';
+
+export default defineConfig( [
+	{
+		ignores: [
+			'coverage/**',
+			'dist/**',
+			'release/**'
+		]
+	},
+
+	{
+		extends: ckeditor5Config,
+
+		languageOptions: {
+			ecmaVersion: 'latest',
+			sourceType: 'module',
+			globals: {
+				...globals.browser
+			}
+		},
+
+		linterOptions: {
+			reportUnusedDisableDirectives: 'warn',
+			reportUnusedInlineConfigs: 'warn'
+		},
+
+		plugins: {
+			'ckeditor5-rules': ckeditor5Rules
+		},
+
+		rules: {
+			'ckeditor5-rules/license-header': [ 'error', { headerLines: [
+				'/**',
+				' * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.',
+				' * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-licensing-options',
+				' */'
+			] } ],
+			'ckeditor5-rules/require-file-extensions-in-imports': [
+				'error',
+				{
+					extensions: [ '.ts', '.js', '.json' ]
+				}
+			],
+			'ckeditor5-rules/prevent-license-key-leak': 'error',
+			'vue/multi-word-component-names': 'off',
```

Comment:
This rule here is unused, because this configuration object does not target any `*.vue` files.

The `'vue/multi-word-component-names': 'off'` is already defined below.